cosmos/test/test_user.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+
+from cosmos.models.user import Profile, User
+
+
+class UserTestCase(TestCase):
+    def test_profile_creation(self):
+        user = User.objects.create(username="potato")
+        # Check that a Profile instance has been crated
+        self.assertIsInstance(user.profile, Profile)
+        # Call the save method of the user to activate the signal
+        # again, and check that it doesn't try to create another
+        # profile instance
+        user.save()
+        self.assertIsInstance(user.profile, Profile)
```

Comment:
Could you add some more tests? For example:
- user creation with all inputs, and that it cannot be made without the required fields
- user creation with invalid input, too long, weird characters, etc.
- user creation with name conflict
- updating of user with the same things as above
The stuff that comes from auth should already have tests by them, so no need to test those, however we do need to test our own stuff thoroughly.

Reply:
I will try adding this over the weekend, unless someone will beat me to it.